model/src/main/proto/survey.proto

```diff
@@ -0,0 +1,182 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Corresponds to a single survey.
+message Survey{
+  // The ID of the survey.
+  string survey_id = 1;
+
+  // A list of questions that make up the survey.
+  repeated SurveyQuestion questions = 2;
+}
+
+// Represents a question that is part of the Survey.
+message SurveyQuestion{
+  // The ID of the survey question.
+  string question_id = 1;
+
+  // The type of question.
+  QuestionType question_type = 2;
+
+  // The actual text value of the question.
+  string  question = 3;
+
+  // The semantic name of the question used to identify it in the analytics dashboard.
+  QuestionName question_name = 4;
+
+  // A list of multi-choice answer options for a given question.
+  repeated  SurveyQuestionAnswer answer_list = 5;
+}
+
+// The list of types a question can have that is used to determine how it is presented by the UI.
+enum QuestionType{
+  // The type of question is unknown.
+  QUESTION_TYPE_UNSPECIFIED=0;
+  // Corresponds to a question with multiple choices.
+  MULTIPLE_CHOICE = 1;
+
+  // Corresponds to a question with multiple choices and a free-text 'other' option.
+  MULTIPLE_CHOICE_WITH_OTHER = 2;
+
+  // Corresponds to a question which takes a free-text response.
+  FREE_FORM = 3;
+}
+
+// Corresponds to the semantic name given to each question.
+enum QuestionName{
+  // The question name is unknown.
+   QUESTION_NAME_UNSPECIFIED = 0;
+  // Corresponds to the name of the user type question.
+  QUESTION_USER_TYPE = 1;
+
+  //  Corresponds to the market fit question.
+  QUESTION_MARKET_FIT = 2;
+
+  // Corresponds to  the NPS question.
+  QUESTION_NPS = 3;
+
+  // Corresponds to  the open-ended question for a NPS promoter.
+  QUESTION_PROMOTER_FEEDBACK = 4;
+
+  // Corresponds to  the open-ended question for a NPS passive.
+  QUESTION_PASSIVE_FEEDBACK = 5;
+
+  // Corresponds to  the open-ended question for a NPS detractor.
+  QUESTION_DETRACTOR_FEEDBACK = 6;
+}
+
+// Represents the multiple choice answer available for a question.
+message SurveyQuestionAnswer{
```

Comment:
I don't think this name well conveys that it represents a multiple choice option--I suggest renaming.

Reply:
Renamed to `SurveyAnswerOption`